x/exchange/spec/01_concepts.md

```diff
@@ -309,6 +353,54 @@ The buyer settlement ratio fee should be added to the buyer settlement flat fee
 The ratio and flat fees can be in any denoms allowed by the market, and do not have to be the same.
 
 
+### Commitment Fees
+
+A market can collect commitment fees at commitment creation and/or during settlement.
+
+A market is also charged an extra Tx fee when doing a settlement.
+
+#### Commitment Creation Fees
+
+Similar to order creation fees, these are paid by the committer at the time of committal.
+A portion of these fees are given to the exchange.
+This can only be defined as a flat fee: `fee_create_commitment_flat`: The available `Coin` fee options that are paid by the committer when committing funds.
+The commitment's `creation_fee` must be at least one of the available `fee_create_commitment_flat` options.
+
+#### Commitment Settlement Fee Collection
+
+During a commitment settlement, a market can collect fees by populating the `fees` field in the settlement msg.
```

Comment:
Message is incorrectly spelled.

During a commitment settlement, a market can collect fees by populating the `fees` field in the settlement `message`.

Reply:
Changed it to specify (and link to) the actual Msg.